src/components/Receive/AddressView.js

```diff
@@ -41,7 +41,7 @@ const AddressView = ({
     <TouchableOpacity activeOpacity={0.5} onPress={openDetails}>
       <View style={styles.container}>
         <View style={styles.addressContainer}>
-          <Text secondary={isUsed} small bold style={index}>{`/${index}`}</Text>
```

Comment:
Here I fix flow error:
```
Cannot create Text element because in property style:
 • Either number [1] is incompatible with boolean literal false [2].
 • Or number [1] is incompatible with empty string [3].
```